src/system/process.py

```diff
@@ -34,6 +35,7 @@ def start(self, command: str, cwd: str) -> None:
             stdout=self.stdout,
             stderr=self.stderr,
         )
+        time.sleep(1)
```

Comment:
Hi @dblock,

I see that the output of the subprocess is not redirected to stdout which is leaving the file empty.So tried by giving it some time to write into stdout before we read it in stdout_data, even a time lapse of 0.1 works here.

And the file pointer is at the end of the file when we tried reading it in stdout_data, so got it to starting using seek().

Reply:
I think instead of adding a delay of fixed 1 second, you can try to check if there is data in stdout before reading.

If someone use an extremely slow server such as t3.small or worse instance, the assumption of 0.1/1 second might not be valid anymore.

It is generally better to do a periodic check in my opinion on the data availability here.

Reply:
I am not sure what problem we're trying to solve in this code. If the process runs and never has any output, then waiting for output will loop indefinitely.  

This method starts a process. It should not have any other side effects.